src/main/java/org/akaza/openclinica/service/ValidateServiceImpl.java

```diff
@@ -326,6 +328,20 @@ public boolean hasCRFpermissionTag(EventDefinitionCrf edc,List<String> permissio
     	
     }
     
+    public boolean hasFormAccess(EventDefinitionCrf edc,List<String> permissionTagsList) {
```

Comment:
Isn't there a method like this in PermissionService? Even if there isn't,  move this method to that class so that all the permission tag validation is in one place

Reply:
done -- move the new method to PermissionService